ballerina/types.bal

```diff
@@ -114,20 +114,6 @@ public distinct class VarrayValue {
     }
 }
 
-
-# Represents XML Oracle DB field.
-#
-# + value - Value of parameter passed into the SQL statement
-public distinct class XmlValue {
```

Comment:
Any reason to remove this type?

Reply:
This PR is removed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/7/commits/cfe79afd0ea4a9fc4d3242c92c526169d2cfa15d

as it was in a seperate PR from gayal but we could not merge that PR because there a lot of conflicts. 
https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/3

Reply:
@lahirusamith but we want to find out why gayal removed this code. As per his comment in his PR `Oracle JDBC does not provide functionality for XML.`. shall we create an issue, look into it further.

Reply:
issue is created https://github.com/ballerina-platform/ballerina-standard-library/issues/1695